scripts/generate_json_docs.py

```diff
@@ -295,11 +296,14 @@ def build_type(type_id, title, contents):
 
 
 def clean_source_path(source):
```

Comment:
`some_module.__name__` provides the class name or the module import name.

```python
>>> from gcloud.datastore import Client
>>> Client.__name__
'Client'
```
or with a module instead of a class...
```python
>>> from gcloud.datastore import client
>>> client.__name__
'gcloud.datastore.client'
```

If it's a class or a module, I need the relative path from gcloud onward. `gcloud/datastore/client.py` etc...